### ISSUE TITLE
Strange/wrong size reported after a lot of samples 

### ISSUE THREAD

**Issue by broetchenrackete36** [CyberShadow/btdu#1]
I have a raid5 array with more than 16TB data on it. At first, btdu shows numbers that look really good but if I let it run quite long (a few hours or so), the reported size is wrong completely (sometimes only a few GB).

```
btdu v0.2.0 @ /btrfs                                                              
--- / -----------------------------------------------------------------------------
  ~167.1 GiB [          ] /<RAID1>                                                  
  ~33.3 GiB [##########] /<RAID5>                                                 

--- Details: ----------------------------------------------------------------------
- Full path: /btrfs/                                                               
- Size: ~200.5 GiB (46054422 samples)                                              
- Average query duration: 10 ms, 723 μs, and 4 hnsecs                              

--- Explanation:                                                                   
Welcome to btdu. You are in the hierarchy root; results will be arranged according 
to their block group and profile, and then by path.                                

Use the arrow keys to navigate, press ? for help.                                  

Samples: 46054422  Resolution: ~403.8 KiB
```

**Comment by CyberShadow** [comment]
Wow, a 64-bit integer overflow.

**Comment by CyberShadow** [comment]
Released v0.2.1. Thanks for the report :)
